fxapom/pages/page.py

```diff
@@ -34,7 +33,7 @@ def is_element_present(self, *locator):
             return False
         finally:
             # set the implicit wait back
-            self.selenium.implicitly_wait(self.testsetup.default_implicit_wait)
+            self.selenium.implicitly_wait(self.timeout)
```

Comment:
Yes, I know. I was trying to be "good" and not include too many changes in a single PR. The idea was to only do what was absolutely necessary to get this running with pytest-selenium and then do another PR afterward which addresses implicit waits and some other page object changes. Do you think it's okay to leave this as is for this PR?